### ISSUE TITLE
(Input) Analog Stick doesn't work on MacOS

### ISSUE THREAD

**Issue by Mjx121418** [libretro/RetroArch#6893]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

[Description of the bug]

### Expected behavior

[What you expected to happen]

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: [The operating system you're running]
- Compiler: [In case you are running local builds]

**Comment by RobLoach** [comment]
Could you fill in the details here? What controller are you using? What version of RA?

**Comment by Mjx121418** [comment]
I'm so sorry for not leaving any detials here.😂

I used retroarch 1.7.3 on macOS 10.13.5. I haved assigned the left analog stick to the up,down,left and right keys on the keyboard of my MacBook. But it doesn't work when I start n64 games such as Super Mario 64. The other buttons works well.

I don't live in a English speaking coutry. Please forgive my poor English.🙂

**Comment by RobLoach** [comment]
What controller? Perhaps look at https://github.com/libretro/retroarch-joypad-autoconfig ?

**Comment by zoltanvb** [comment]
Very old ticket, but anyways now I tested this on MacOS by assigning cursor keys to left analog stick directions. It works properly with Remote Retropad, so I suspect the original problem may have been with the core settings, or solved in the meantime. I believe this can be closed.

**Comment by warmenhoven** [comment]
I agree, this would have been filed before the mfi and sdl2 drivers were available. Analog sticks should also work correctly with the hid driver now.
